hack/generate.sh

```diff
@@ -33,8 +33,9 @@ function generate_python_client() {
         -v "${__root}"/tools/generate_python_client.sh:/script.sh:ro,Z \
         -e SWAGGER_FILE=/swagger.yaml -e OUTPUT=/local/assisted-service-client/ \
         quay.io/ocpmetal/swagger-codegen-cli:2.4.15 /script.sh
-     cd "${dest}"/assisted-service-client/ && python3 setup.py sdist --dist-dir "${dest}"
      cd "${dest}"/assisted-service-client/ && python3 "${__root}"/tools/client_package_initializer.py "${dest}"/assisted-service-client/  https://github.com/openshift/assisted-service --build
+     rm -f "${dest}"/assisted-service-client-*.tar.gz
```

Comment:
I don't understand
Why do we create it in the first place if we're not using it?

Reply:
We don't need that anymore. Removed that line.